docs/reference/cluster/stats.asciidoc

```diff
@@ -1307,6 +1307,142 @@ Each repository type may also include other statistics about the repositories of
 
 ====
 
+`ccs`::
+(object) Contains information relating to <<modules-cross-cluster-search, cross-cluster search>> settings and activity in the cluster.
++
+.Properties of `ccs`
+[%collapsible%open]
+=====
+
+
+`_search`:::
+(object) Contains telemetry information about the <<modules-cross-cluster-search, cross-cluster searches>> in the cluster.
++
+.Properties of `_search`
+[%collapsible%open]
+======
+`total`:::
+(integer) The total number of cross-cluster search requests that have been executed by the cluster.
```

Comment:
There is a macro (?, not sure the right word) that the docs team set up to ensure that "cross-cluster search" is always expressed the same way in the docs, so use that in places like this. The macro is `{ccs}`. You can see examples in the `search-across-clusters.asciidoc` file.